src/routing.ts

```diff
@@ -1,39 +1,46 @@
-import { IRouter } from "express";
+import { IRouter, RequestHandler } from "express";
 import { CommonConfig } from "./config-type";
+import { ContentType } from "./content-type";
 import { DependsOnMethod } from "./depends-on-method";
 import { AbstractEndpoint } from "./endpoint";
 import { AbstractLogger } from "./logger";
+import { metaSymbol } from "./metadata";
 import { walkRouting } from "./routing-walker";
 import { ServeStatic } from "./serve-static";
+import { LocalResponse } from "./server-helpers";
 
 export interface Routing {
   [SEGMENT: string]: Routing | DependsOnMethod | AbstractEndpoint | ServeStatic;
 }
 
+export type Parsers = Record<ContentType, RequestHandler[]>;
+
 export const initRouting = ({
   app,
   rootLogger,
   config,
   routing,
+  parsers,
 }: {
   app: IRouter;
   rootLogger: AbstractLogger;
   config: CommonConfig;
   routing: Routing;
+  parsers?: Parsers;
 }) =>
   walkRouting({
     routing,
     hasCors: !!config.cors,
     onEndpoint: (endpoint, path, method, siblingMethods) => {
-      app[method](path, async (request, response) => {
-        const logger = config.childLoggerProvider
-          ? await config.childLoggerProvider({ request, parent: rootLogger })
-          : rootLogger;
-        logger.info(`${request.method}: ${path}`);
+      const middlewares = parsers?.[endpoint.getRequestType()] || [];
+      if (middlewares.length) {
+        app.use(path, middlewares);
```

Comment:
From the `app.use` documentation:

> A route will match any path that follows its path immediately with a “/”. For example: app.use('/apple', ...) will match “/apple”, “/apple/images”, “/apple/images/news”, and so on.

So... probably this will not work as good as I wanted to.

Reply:
fixed in e2b3ad87